diff --git a/index.js b/index.js
index 6d5d2a7891ba13e30c581aa82da1f86a49f5fa36..adb674864cb0da258682fe583ac3c8504573c6ef 100644
--- a/index.js
+++ b/index.js
@@ -1,6 +1,6 @@
-var convert = require('color-convert');
+import convert from 'color-convert';
 
-module.exports = function (cstr) {
+export default function (cstr) {
     var m, conv, parts, alpha;
     if (m = /^((?:rgb|hs[lv]|cmyk|xyz|lab)a?)\s*\(([^\)]*)\)/.exec(cstr)) {
         var name = m[1];
