diff --git a/node_modules/scratch-paint/.bun-tag-44c853d08dbc22b7 b/.bun-tag-44c853d08dbc22b7
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/src/components/button/button.jsx b/src/components/button/button.jsx
index 7f1c1660cdbe984917e3a3b35e6c343f7620890f..908ebc068b2ded2c30ee275e67cedfa4dbadbae2 100644
--- a/src/components/button/button.jsx
+++ b/src/components/button/button.jsx
@@ -9,7 +9,7 @@ import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import styles from './button.css';
+import styles from './button.module.css';
 
 const ButtonComponent = ({
     className,
diff --git a/src/components/button/button.css b/src/components/button/button.module.css
similarity index 100%
rename from src/components/button/button.css
rename to src/components/button/button.module.css
diff --git a/src/components/button-group/button-group.jsx b/src/components/button-group/button-group.jsx
index 3c5a316a6042e81f2fc9bc7247c8d6d377f367a8..d2d027cb9cf647a0d9cbd656c6dec2491fb1602f 100644
--- a/src/components/button-group/button-group.jsx
+++ b/src/components/button-group/button-group.jsx
@@ -2,7 +2,7 @@ import classNames from 'classnames';
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import styles from './button-group.css';
+import styles from './button-group.module.css';
 
 const ButtonGroup = props => (
     <div className={classNames(props.className, styles.buttonGroup)}>
diff --git a/src/components/button-group/button-group.css b/src/components/button-group/button-group.module.css
similarity index 100%
rename from src/components/button-group/button-group.css
rename to src/components/button-group/button-group.module.css
diff --git a/src/components/color-button/color-button.jsx b/src/components/color-button/color-button.jsx
index 07d9b8a785d9134504d0787a12895ec68e72d4cb..1cbd1b5474f01e44613946e31a43ab9b135c5318 100644
--- a/src/components/color-button/color-button.jsx
+++ b/src/components/color-button/color-button.jsx
@@ -6,7 +6,7 @@ import {MIXED} from '../../helper/style-path';
 
 import noFillIcon from './no-fill.svg';
 import mixedFillIcon from './mixed-fill.svg';
-import styles from './color-button.css';
+import styles from './color-button.module.css';
 import GradientTypes from '../../lib/gradient-types';
 import log from '../../log/log';
 
diff --git a/src/components/color-button/color-button.css b/src/components/color-button/color-button.module.css
similarity index 100%
rename from src/components/color-button/color-button.css
rename to src/components/color-button/color-button.module.css
diff --git a/src/components/color-picker/color-picker.jsx b/src/components/color-picker/color-picker.jsx
index 398c5cc6d8759909644f3c1e0f4a9f70f4d36398..dfc8f67ec123630895411a1db92337012baf00e5 100644
--- a/src/components/color-picker/color-picker.jsx
+++ b/src/components/color-picker/color-picker.jsx
@@ -7,7 +7,7 @@ import parseColor from 'parse-color';
 
 import Slider, {CONTAINER_WIDTH, HANDLE_WIDTH} from '../forms/slider.jsx';
 import LabeledIconButton from '../labeled-icon-button/labeled-icon-button.jsx';
-import styles from './color-picker.css';
+import styles from './color-picker.module.css';
 import GradientTypes from '../../lib/gradient-types';
 import {MIXED} from '../../helper/style-path';
 
diff --git a/src/components/color-picker/color-picker.css b/src/components/color-picker/color-picker.module.css
similarity index 100%
rename from src/components/color-picker/color-picker.css
rename to src/components/color-picker/color-picker.module.css
diff --git a/src/components/coming-soon/coming-soon.jsx b/src/components/coming-soon/coming-soon.jsx
index fa9285b0cd9d265d3a2a7753d6b0fb13d97b851a..b9550aad28e69d23051ef378f6a88cbcf6748825 100644
--- a/src/components/coming-soon/coming-soon.jsx
+++ b/src/components/coming-soon/coming-soon.jsx
@@ -9,7 +9,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import ReactTooltip from 'react-tooltip';
 
-import styles from './coming-soon.css';
+import styles from './coming-soon.module.css';
 
 import awwCatIcon from './aww-cat.png';
 import coolCatIcon from './cool-cat.png';
diff --git a/src/components/coming-soon/coming-soon.css b/src/components/coming-soon/coming-soon.module.css
similarity index 100%
rename from src/components/coming-soon/coming-soon.css
rename to src/components/coming-soon/coming-soon.module.css
diff --git a/src/components/dropdown/dropdown.jsx b/src/components/dropdown/dropdown.jsx
index a8ec3f95f2730fc3eacbb8bf17acd75f575c668c..62d2c29b3043bfcbc621d81c99606e0d3f8e3825 100644
--- a/src/components/dropdown/dropdown.jsx
+++ b/src/components/dropdown/dropdown.jsx
@@ -4,7 +4,7 @@ import Popover from 'react-popover';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import styles from './dropdown.css';
+import styles from './dropdown.module.css';
 
 import dropdownIcon from './dropdown-caret.svg';
 
diff --git a/src/components/dropdown/dropdown.css b/src/components/dropdown/dropdown.module.css
similarity index 100%
rename from src/components/dropdown/dropdown.css
rename to src/components/dropdown/dropdown.module.css
diff --git a/src/components/fixed-tools/fixed-tools.jsx b/src/components/fixed-tools/fixed-tools.jsx
index b1d325fa5c4705eb1c19eb21bc0c47f119287fe1..f0e636dbd0ba901b4221b47dfdd05d39b774424e 100644
--- a/src/components/fixed-tools/fixed-tools.jsx
+++ b/src/components/fixed-tools/fixed-tools.jsx
@@ -19,7 +19,7 @@ import Label from '../forms/label.jsx';
 import LabeledIconButton from '../labeled-icon-button/labeled-icon-button.jsx';
 import layout from '../../lib/layout-constants';
 import {hideLabel} from '../../lib/hide-label';
-import styles from './fixed-tools.css';
+import styles from './fixed-tools.module.css';
 
 import groupIcon from './icons/group.svg';
 import redoIcon from './icons/redo.svg';
diff --git a/src/components/fixed-tools/fixed-tools.css b/src/components/fixed-tools/fixed-tools.module.css
similarity index 100%
rename from src/components/fixed-tools/fixed-tools.css
rename to src/components/fixed-tools/fixed-tools.module.css
diff --git a/src/components/font-dropdown/font-dropdown.jsx b/src/components/font-dropdown/font-dropdown.jsx
index 08a827a9bfcec2551b4f7b9019da88f2a012ca24..d9c3611220cff57e3005225706f7656d15187afe 100644
--- a/src/components/font-dropdown/font-dropdown.jsx
+++ b/src/components/font-dropdown/font-dropdown.jsx
@@ -6,7 +6,7 @@ import Button from '../button/button.jsx';
 import Dropdown from '../dropdown/dropdown.jsx';
 import InputGroup from '../input-group/input-group.jsx';
 import Fonts from '../../lib/fonts';
-import styles from './font-dropdown.css';
+import styles from './font-dropdown.module.css';
 
 const ModeToolsComponent = props => (
     <Dropdown
diff --git a/src/components/font-dropdown/font-dropdown.css b/src/components/font-dropdown/font-dropdown.module.css
similarity index 100%
rename from src/components/font-dropdown/font-dropdown.css
rename to src/components/font-dropdown/font-dropdown.module.css
diff --git a/src/components/forms/input.jsx b/src/components/forms/input.jsx
index 3a5feb13a33f1060fba28cadd8e1c678cb18257e..f6e51b60aed218e1f684e7728016e6c3fc9d955a 100644
--- a/src/components/forms/input.jsx
+++ b/src/components/forms/input.jsx
@@ -10,7 +10,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 
-import styles from './input.css';
+import styles from './input.module.css';
 
 const Input = props => {
     const {small, range, ...componentProps} = props;
diff --git a/src/components/forms/input.css b/src/components/forms/input.module.css
similarity index 100%
rename from src/components/forms/input.css
rename to src/components/forms/input.module.css
diff --git a/src/components/forms/label.jsx b/src/components/forms/label.jsx
index 475ffb00988bb8e5d0875fda1c8e77133a4f0bbb..d48efe6fa4ea116f5db2698eae51598d57585c31 100644
--- a/src/components/forms/label.jsx
+++ b/src/components/forms/label.jsx
@@ -5,7 +5,7 @@ See https://github.com/LLK/scratch-paint/issues/13 */
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import styles from './label.css';
+import styles from './label.module.css';
 
 const Label = props => (
     <label className={styles.inputGroup}>
diff --git a/src/components/forms/label.css b/src/components/forms/label.module.css
similarity index 94%
rename from src/components/forms/label.css
rename to src/components/forms/label.module.css
index e55c4231ddfda4075633aa61d2badd99c48cb075..aeb35b90ffa437e5800ece8a401efaaf5776226c 100644
--- a/src/components/forms/label.css
+++ b/src/components/forms/label.module.css
@@ -4,7 +4,7 @@ See https://github.com/LLK/scratch-paint/issues/13 */
 
 @import "../../css/units.css";
 @import "../../css/colors.css";
-@import "../input-group/input-group.css";
+@import "../input-group/input-group.module.css";
 
 .input-label, .input-label-secondary {
     font-size: 0.625rem;
diff --git a/src/components/forms/slider.jsx b/src/components/forms/slider.jsx
index 603fb7f74b2b43dc2699dd173566b979e752fde3..fae0c4edd33b7706daf617b03e21253cdb874c56 100644
--- a/src/components/forms/slider.jsx
+++ b/src/components/forms/slider.jsx
@@ -4,7 +4,7 @@ import bindAll from 'lodash.bindall';
 import classNames from 'classnames';
 import {getEventXY} from '../../lib/touch-utils';
 
-import styles from './slider.css';
+import styles from './slider.module.css';
 
 const CONTAINER_WIDTH = 150;
 const HANDLE_WIDTH = 26;
diff --git a/src/components/forms/slider.css b/src/components/forms/slider.module.css
similarity index 100%
rename from src/components/forms/slider.css
rename to src/components/forms/slider.module.css
diff --git a/src/components/input-group/input-group.jsx b/src/components/input-group/input-group.jsx
index 0d09ad93697ef8398665b2e8534920cf0b3aa855..ce8c2dedb072179472e698ee38a3f1fdaa5389f8 100644
--- a/src/components/input-group/input-group.jsx
+++ b/src/components/input-group/input-group.jsx
@@ -2,7 +2,7 @@ import classNames from 'classnames';
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import styles from './input-group.css';
+import styles from './input-group.module.css';
 
 const InputGroup = props => (
     <div
diff --git a/src/components/input-group/input-group.css b/src/components/input-group/input-group.module.css
similarity index 100%
rename from src/components/input-group/input-group.css
rename to src/components/input-group/input-group.module.css
diff --git a/src/components/labeled-icon-button/labeled-icon-button.jsx b/src/components/labeled-icon-button/labeled-icon-button.jsx
index bdf541f07a7fbbd20a9a61eb677f6141921903a6..4d168cfa4d9fa5c6334d8698ecc48fd1635c49e2 100644
--- a/src/components/labeled-icon-button/labeled-icon-button.jsx
+++ b/src/components/labeled-icon-button/labeled-icon-button.jsx
@@ -8,7 +8,7 @@ import PropTypes from 'prop-types';
 
 import Button from '../button/button.jsx';
 
-import styles from './labeled-icon-button.css';
+import styles from './labeled-icon-button.module.css';
 
 const LabeledIconButton = ({
     className,
diff --git a/src/components/labeled-icon-button/labeled-icon-button.css b/src/components/labeled-icon-button/labeled-icon-button.module.css
similarity index 100%
rename from src/components/labeled-icon-button/labeled-icon-button.css
rename to src/components/labeled-icon-button/labeled-icon-button.module.css
diff --git a/src/components/loupe/loupe.jsx b/src/components/loupe/loupe.jsx
index 8aadbf38d62c2527a9a1c653f1e0984011a660fd..719dbe44e9d9736873695d8d55b033006d193bd1 100644
--- a/src/components/loupe/loupe.jsx
+++ b/src/components/loupe/loupe.jsx
@@ -6,7 +6,7 @@ import Box from '../box/box.jsx';
 
 import {LOUPE_RADIUS, ZOOM_SCALE} from '../../helper/tools/eye-dropper';
 
-import styles from './loupe.css';
+import styles from './loupe.module.css';
 
 class LoupeComponent extends React.Component {
     constructor (props) {
diff --git a/src/components/loupe/loupe.css b/src/components/loupe/loupe.module.css
similarity index 100%
rename from src/components/loupe/loupe.css
rename to src/components/loupe/loupe.module.css
diff --git a/src/components/mode-tools/mode-tools.jsx b/src/components/mode-tools/mode-tools.jsx
index 78771edb8e95f87c7fecb7c7e1ef73953b8830c2..c3e9b9925c9e9255636cb70eebc72c780894853d 100644
--- a/src/components/mode-tools/mode-tools.jsx
+++ b/src/components/mode-tools/mode-tools.jsx
@@ -19,7 +19,7 @@ import LabeledIconButton from '../labeled-icon-button/labeled-icon-button.jsx';
 import Modes from '../../lib/modes';
 import Formats, {isBitmap, isVector} from '../../lib/format';
 import {hideLabel} from '../../lib/hide-label';
-import styles from './mode-tools.css';
+import styles from './mode-tools.module.css';
 
 import copyIcon from './icons/copy.svg';
 import pasteIcon from './icons/paste.svg';
diff --git a/src/components/mode-tools/mode-tools.css b/src/components/mode-tools/mode-tools.module.css
similarity index 100%
rename from src/components/mode-tools/mode-tools.css
rename to src/components/mode-tools/mode-tools.module.css
diff --git a/src/components/paint-editor/paint-editor.jsx b/src/components/paint-editor/paint-editor.jsx
index c94d35853c23a2ce754f9f9fc764866ad8f74882..45ac50f6dbb39478f249864c12a572fa96181451 100644
--- a/src/components/paint-editor/paint-editor.jsx
+++ b/src/components/paint-editor/paint-editor.jsx
@@ -35,7 +35,7 @@ import StrokeWidthIndicatorComponent from '../../containers/stroke-width-indicat
 import TextMode from '../../containers/text-mode.jsx';
 
 import Formats, {isBitmap, isVector} from '../../lib/format';
-import styles from './paint-editor.css';
+import styles from './paint-editor.module.css';
 
 import bitmapIcon from './icons/bitmap.svg';
 import zoomInIcon from './icons/zoom-in.svg';
diff --git a/src/components/paint-editor/paint-editor.css b/src/components/paint-editor/paint-editor.module.css
similarity index 100%
rename from src/components/paint-editor/paint-editor.css
rename to src/components/paint-editor/paint-editor.module.css
diff --git a/src/components/scrollable-canvas/scrollable-canvas.jsx b/src/components/scrollable-canvas/scrollable-canvas.jsx
index 236a54032aac5679d876babb56b2edb356b55657..92b40f736af7a39d0be931475dca95e3ddf0dba1 100644
--- a/src/components/scrollable-canvas/scrollable-canvas.jsx
+++ b/src/components/scrollable-canvas/scrollable-canvas.jsx
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import styles from './scrollable-canvas.css';
+import styles from './scrollable-canvas.module.css';
 
 const ScrollableCanvasComponent = props => (
     <div
diff --git a/src/components/scrollable-canvas/scrollable-canvas.css b/src/components/scrollable-canvas/scrollable-canvas.module.css
similarity index 100%
rename from src/components/scrollable-canvas/scrollable-canvas.css
rename to src/components/scrollable-canvas/scrollable-canvas.module.css
diff --git a/src/components/tool-select-base/tool-select-base.jsx b/src/components/tool-select-base/tool-select-base.jsx
index d9b2be003fe7f8a870f947cec50c7c6091d5922f..70af2c7a5ad81ffd60ace065b4d630fa3acdd142 100644
--- a/src/components/tool-select-base/tool-select-base.jsx
+++ b/src/components/tool-select-base/tool-select-base.jsx
@@ -5,7 +5,7 @@ import {injectIntl, intlShape} from 'react-intl';
 
 import Button from '../button/button.jsx';
 
-import styles from './tool-select-base.css';
+import styles from './tool-select-base.module.css';
 
 const ToolSelectComponent = props => (
     <Button
diff --git a/src/components/tool-select-base/tool-select-base.css b/src/components/tool-select-base/tool-select-base.module.css
similarity index 100%
rename from src/components/tool-select-base/tool-select-base.css
rename to src/components/tool-select-base/tool-select-base.module.css
diff --git a/src/containers/font-dropdown.jsx b/src/containers/font-dropdown.jsx
index f0eecdfa98c01ba5cbce85c7adb7b7736dd4a529..b4fab234a39caae7db93a65f57034d64ab5f94a7 100644
--- a/src/containers/font-dropdown.jsx
+++ b/src/containers/font-dropdown.jsx
@@ -8,7 +8,7 @@ import FontDropdownComponent from '../components/font-dropdown/font-dropdown.jsx
 import Fonts from '../lib/fonts';
 import {changeFont} from '../reducers/font';
 import {getSelectedLeafItems} from '../helper/selection';
-import styles from '../components/font-dropdown/font-dropdown.css';
+import styles from '../components/font-dropdown/font-dropdown.module.css';
 
 class FontDropdown extends React.Component {
     constructor (props) {
diff --git a/src/containers/paper-canvas.jsx b/src/containers/paper-canvas.jsx
index b28b40b98b9f7858d9ae39080cbcbd16dfac5ca9..5de57502b1042bbdee7f3137e1b81de51e906f1c 100644
--- a/src/containers/paper-canvas.jsx
+++ b/src/containers/paper-canvas.jsx
@@ -22,7 +22,7 @@ import {changeFormat} from '../reducers/format';
 import {updateViewBounds} from '../reducers/view-bounds';
 import {saveZoomLevel, setZoomLevelId} from '../reducers/zoom-levels';
 
-import styles from './paper-canvas.css';
+import styles from './paper-canvas.module.css';
 
 class PaperCanvas extends React.Component {
     constructor (props) {
diff --git a/src/containers/paper-canvas.css b/src/containers/paper-canvas.module.css
similarity index 100%
rename from src/containers/paper-canvas.css
rename to src/containers/paper-canvas.module.css
